src/contexts/RoomContextState.tsx

```diff
@@ -9,14 +8,16 @@ export const RoomContext = createContext<IRoomContext>({} as IRoomContext)
 
 export const RoomProvider: React.FC = ({ children }) => {
   const [room, setRoom] = useState<any>(null)
-  const [roomKey] = useLocalStorage('room_key', '-MjKz1_1N27ZUvGxfVrA')
+  const [roomKey, setRoomKey] = useState('')
   let unsuscribeEvent: Unsubscribe | null = null
 
   useEffect(() => {
+    console.log('roomKey', roomKey)
+
     unsuscribeEvent = getRoomByKeyWithSync(roomKey, setRoom)
```

Comment:
mmm ahora que lo pienso, creo que aca hace falta un if
```suggestion
if(roomKey) {unsuscribeEvent = getRoomByKeyWithSync(roomKey, setRoom)}
```